src/endpoints/endpoint_registry.cpp

```diff
@@ -132,6 +132,16 @@ namespace ccf::endpoints
     return metrics[method][verb];
   }
 
+  void default_locally_committed_func(
+    CommandEndpointContext& ctx, const TxID& tx_id)
+  {
+    // Only transactions that acquired one or more map handles
+    // have a TxID, while others (e.g. unauthenticated commands)
+    // don't. Also, only report a TxID if the consensus is set, as
+    // the consensus is required to verify that a TxID is valid.
```

Comment:
I think this comment should stay on its original location in `frontend.h`, since its describing the check _before_ `execute_endpoint_locally_committed` is called rather than the behaviour of this default implementation.